components/tooling/detekt/build.gradle

```diff
@@ -17,6 +17,13 @@ dependencies {
     testImplementation Dependencies.tools_detekt_test
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    // Gradle keeps warning about different Kotlin versions being on the classpath. I am unsure
+    // how to resolve that and if it is even resolvable in this cross-library setup. Anyhow,
+    // since it is only a warning I am going to suppress them for this project specifically.
+    kotlinOptions.allWarningsAsErrors = false
```

Comment:
Should we file an issue for this? Although `allWarningsAsErrors` were completely annoying for me in development there are helpful to catch early possible bugs.

Reply:
They are still enabled everywhere else. This is here is for `tooling-detekt`.

Reply:
👍